assets/styles.css

```diff
@@ -1,3 +1,409 @@
-.header {
- color: red;
-}
\ No newline at end of file
+body {
+  margin: 0;
+  /* font-family: ""; */
+}
+
+a {
+  text-decoration: none;
+}
+
+header {
+  height: 30px;
+  width: 100%;
+  background-color: whitesmoke;
+  position: fixed;
+  top: 0;
+  z-index: 10;
+  display: flex;
+  padding: 5px;
+  border: 1px solid #cccccc;
+  cursor: pointer;
+}
+.container {
+  width: 100%;
+  padding-top: 5px;
+  margin: 0 auto;
+  display: flex;
+  vertical-align: baseline;
+}
+
+.header-left {
+  float: left;
+  display: flex;
+  flex: 0.2;
+  justify-content: space-evenly;
+  /* align-items: center; */
+}
+.svg-icon {
+  height: 20px;
+  width: 20px;
+}
+
+.section-title {
+  width: max-content;
+  font-family: nyt-franklin, helvetica, arial, sans-serif;
+  font-weight: 700;
+  font-size: 14.25px;
+  text-transform: uppercase;
+  letter-spacing: 0.7px;
+  line-height: 19px;
+}
+.section-title a {
+  color: #000;
+}
+.header-right button {
+  background-color: rgb(86, 123, 149);
+  border: rgb(50, 104, 145);
+  color: white;
+  transition: all 0.6s ease 0s;
+  width: max-content;
+  float: left;
+  display: block;
+  cursor: pointer;
+}
+
+.header-right {
+  float: left;
+  display: flex;
+  /* padding: 5px; */
+  margin-right: 5px;
+  width: max-content;
+  flex: 0.2;
+  justify-content: space-evenly;
+  /* align-items: center; */
+}
+.site-logo {
+  margin: 0 auto;
+}
+.articleBody {
+  display: flex;
+  padding-top: 50px;
+  margin: 0 auto;
+  flex-direction: column;
+  align-items: center;
+}
+.article-title {
+  font-family: nyt-cheltenham, georgia, "times new roman", times, serif;
+  font-style: italic;
+  font-weight: 700;
+  font-size: 1.2rem;
+  width: calc(100% - 40px);
+  margin: 0 20px 20px;
+  max-width: 600px;
+}
+.socialToolBar {
+  border-top: 1px solid #ebebeb;
+  display: flex;
+  /* flex-direction: row; */
+  width: 600px;
+  margin: 0 20px 20px;
+  /* align-items: center; */
+  /* max-width: 600px; */
```

Comment:
- [ ]  Commented code is not a good practice in clean coding. Please avoid it.

Reply:
Done.